dist/index.html

```diff
@@ -0,0 +1 @@
+<!doctype html><html><head><meta charset="utf-8"/><title>Webpack App</title><script defer="defer" src="main.js"></script></head><body><main><section><h1>Leaderboard</h1><div id="titleRefresh"><h3 class="">Recent scores</h3><button class="">Refresh</button></div><ul style="list-style: none"><li class="nameScore"><p>Name:</p><p>100</p></li><li class="nameScore"><p>Name:</p><p>20</p></li><li class="nameScore"><p>Name:</p><p>50</p></li></ul></section><section><form action="#" id="form"><h2>Add your score</h2><input name="" id="" placeholder="Your name" style="border: solid"> <input type="number" name="" id="" placeholder="Your score" style="border: solid"> <button type="submit" style="border: solid">Submit</button></form></section></main></body></html>
```

Comment:
:heavy_check_mark: Leaderboard: project setup is implemented as stated in the project requirements. :clap: :clap: 

![Screenshot from 2022-11-16 00-58-17](https://user-images.githubusercontent.com/16392046/202034739-901ecb9e-b06f-4752-a012-938f8944e6de.png)